src/DownloadFile.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Item } from './types';
+
+interface DownloadFileProps {
+  items: Item[];
+}
+
+const DownloadFile = ({ items }: DownloadFileProps) => {
+  function download(content: string, fileName: string, contentType: string) {
```

Comment:
why do you use different type of functions here and below?

Reply:
Done. Thank you.